contracts/RequestResponseConsumer.sol

```diff
@@ -1,10 +1,11 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.16;
 
-import "@bisonai/orakl-contracts/src/v0.1/RequestResponseConsumerBase.sol";
-import "@bisonai/orakl-contracts/src/v0.1/interfaces/RequestResponseCoordinatorInterface.sol";
+import { RequestResponseConsumerFulfillUint128 } from "@bisonai/orakl-contracts/src/v0.1/RequestResponseConsumerFulfill.sol";
+import { RequestResponseConsumerBase } from "@bisonai/orakl-contracts/src/v0.1/RequestResponseConsumerBase.sol";
+import { Orakl } from "@bisonai/orakl-contracts/src/v0.1/libraries/Orakl.sol";
 
-contract RequestResponseConsumer is RequestResponseConsumerBase {
+contract RequestResponseConsumer is RequestResponseConsumerFulfillUint128 {
```

Comment:
Do we need to add a comment here to let consumers know this mockup is for requesting a Unit128, and we have other data types.

Reply:
Good point! Let me emphasize that 🚀